{base_frame: base_link, buadrate: 115200, cmd_vel_topic: cmd_vel, odom_frame: odom,
  odom_topic: wheel_odom, out_pid_debug_enable: false, port: /dev/neuronbot2, publish_tf: false}
